example_code/Makefile

```diff
@@ -24,9 +26,10 @@ all: $(C_BINS) $(F_BINS)
 	$(FC) $(FFLAGS) -o $@ $+
 
 run: $(C_BINS)
-	@for bin in $+; do				\
-	    echo --$$bin------------------------------;	\
-	    $(RUNCMD) $(RUNOPT) ./$$bin || exit $$?;	\
+	@for bin in $+; do					\
+	    echo -- $$bin ------------------------------;	\
+	    $(RUNCMD) $(RUNOPT) ./$$bin;			\
+	    echo -- exit status: $$?;				\
```

Comment:
I think we need to reintroduce a statement to the effect: `if [ $$? -ne 0 ]; then exit $$?; fi;`  Otherwise, Travis will not be able to flag failures.

Reply:
Oh, right, we aren't testing these with Travis during spec builds.  This is because newly introduced APIs aren't yet supported and can't be tested.  This is still useful to have, because I think we do run these tests (e.g. for a specific tagged version of the spec) as part of automated testing.

Reply:
The reason for this change is that `make run` was failing on `shmem_global_exit`, which intentionally returns a nonzero status.

Reply:
Hmm .. we could introduce a `TESTS_XFAIL` list to handle cases like this.  The atomics clarification proposal also introduces three examples with undefined behavior, which we would want to exclude from running.  So, maybe `TESTS_SKIP`?  Or, move to a proper testing harness.